specs/EmergencyAuthCodeEligibleOfficers.yml

```diff
@@ -3,12 +3,15 @@ info:
   title: Companies House Internal Oracle Query API
   version: '1.0'
 servers:
-  - url: http://localhost:18XXX
+  - url: http://localhost:18565
 tags:
   - name: emergency-auth-code-officers
```

Comment:
This file is called EmergencyAuthCodeEligibleOfficers, it doesn't seem right to add our endpoint to this spec.
Can we just create a seperate yaml file for our query?
Something like specs/CompanyActionCode.yml?

Reply:
Oh yes, was going to check with someone about either renaming this file to something more generic or have our endpoint separate. Will do that.

Reply:
Spoke to @jphillips2121 . Will rename the file to make it more generic.